### PR TITLE
startTime, imageQuality ++

### PR DESCRIPTION
**Pull request by magnurud**

### CODE REVIEW

File: lascli/parser/predictions.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
-def post_predictions(las_client: Client, document_id, model_id, auto_rotate, max_pages):
-    return las_client.create_prediction(document_id, model_id, auto_rotate=auto_rotate, max_pages=max_pages)
+def post_predictions(las_client: Client, document_id, model_id, auto_rotate, max_pages, image_quality):
+    return las_client.create_prediction(
+        document_id,
+        model_id,
+        auto_rotate=auto_rotate,
+        max_pages=max_pages,
+        image_quality = image_quality,
```
Comment (augustak) [inline lascli/parser/predictions.py:10]: remove whitespace

File: lascli/parser/predictions.py [hunk 2]
```diff
@@ -18,6 +24,7 @@ def create_predictions_parser(subparsers):
     create_predicton_parser.add_argument('model_id')
     create_predicton_parser.add_argument('--auto-rotate', action='store_true', default=False)
     create_predicton_parser.add_argument('--max-pages', type=int, default=1)
+    create_predicton_parser.add_argument('--image-quality', type=str, default='low')
```
Comment (augustak) [inline lascli/parser/predictions.py:27]: use choices={'LOW', 'HIGH'}

File: lascli/parser/transitions.py [hunk 3]
```diff
@@ -135,6 +141,12 @@ def create_transitions_parser(subparsers):
     update_execution_parser.add_argument('status', choices=['succeeded', 'failed', 'rejected', 'retry'])
     update_execution_parser.add_argument('--output_path', '-o')
     update_execution_parser.add_argument('--error_path', '-e')
+    update_execution_parser.add_argument('--start-time')
```
Comment (augustak) [inline lascli/parser/transitions.py:144]: I think we should make sending start time easier if possible. It is cumbersome typing in an ISO date string. Maybe use datetime somehow?

File: lascli/parser/transitions.py [hunk 4]
```diff
@@ -135,6 +141,12 @@ def create_transitions_parser(subparsers):
...
     update_execution_parser.set_defaults(cmd=update_transition_execution)
 
+    heartbeat_parser = subparsers.add_parser('heartbeat')
+    heartbeat_parser.add_argument('transition_id')
+    heartbeat_parser.add_argument('execution_id')
+    heartbeat_parser.set_defaults(cmd=send_heartbeat)
```
Comment (augustak) [inline lascli/parser/transitions.py:150]: heartbeats_parser to follow naming convention?

**Review by augustak — CHANGES_REQUESTED** [review]

File: lascli/parser/transitions.py (hunk 3 above)
Comment (magnurud) [reply lascli/parser/transitions.py:144]: I don't see when people will actually use this unless they have the exact time...

Comment (magnurud) [reply lascli/parser/transitions.py:144]: I don't really think our users will use this feature in the CLI, so I suggest we leave it like this until we get a pushback.

**Review by augustak — APPROVED** [review]

**Pull request merged**